Source/SA/DPD/DPD.cs

```diff
@@ -0,0 +1,207 @@
+﻿using NationalInstruments.RFmx.SpecAnMX;
+using NationalInstruments.ModularInstruments.NIRfsg;
+using static NationalInstruments.ReferenceDesignLibraries.SG;
+using System;
+
+namespace NationalInstruments.ReferenceDesignLibraries.SA
+{
+    public static class DPD
+    {
+        #region Type_Definitionss
+        public struct CommonConfiguration
+        {
+            public double DpdMeasurementInterval;                                                  //Seconds
+            public RFmxSpecAnMXDpdMeasurementSampleRateMode DpdSampleRateMode;
```

Comment:
If we remove the sample rate and just use the waveform struct's sample rate as @smooresni  and I have suggested, then I believe we should also remove this property as well (since there won't be an input for a user to set the sample rate anyway). 
```suggestion
```

Reply:
@mdwhitten , the sample rate is the user input. If the user selects DPD sample rate mode to user, then the sample rate applies. I recommend we change this variable to UsersSampleRate_Hz instead. Thoughts?

Reply:
Agreed this property needs to stay. I think it's okay to keep it at SampleRate_Hz with potentially a comment to clarify it is ignored unless SampleRateMode is set to user.

Reply:
I guess that's my point - is there any reason we need to support a user-input sample rate that is different from the waveform's sample rate? I don't know DPD at all so I am just wondering.

Reply:
@mdwhitten this is referring to the sample rate used on the analyzer side to capture the distorted waveform. i haven't seen custom sample rates used too much in practice. @heyItsDale any thoughts on that?

Reply:
Decision made with @heyItsDale / @smooresni / @cindyxie to remove the property and set sample rate mode to "Reference Waveform" explicitly. Sample rate will be set by the waveform.